TurkishFilter/sections/general_extensions.txt

```diff
@@ -32,6 +32,8 @@ technopat.net#%#(function() { var w_open = window.open, regex = /goo.gl/; window
 ! dizipal*.com - frequently changed domains
 dizipal228.com,dizipal229.com,dizipal230.com,dizipal231.com,dizipal232.com,dizipal233.com,dizipal234.com,dizipal235.com,dizipal236.com,dizipal237.com,dizipal238.com,dizipal239.com,dizipal240.com,dizipal241.com,dizipal242.com,dizipal243.com,dizipal244.com,dizipal245.com,dizipal246.com,dizipal247.com,dizipal248.com,dizipal249.com,dizipal250.com#%#//scriptlet("remove-attr", "data-money", "div")
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/117668
+filmizlew.org#%#//scriptlet('set-cookie', 'dadadaaaa', '1')
```

Comment:
better this
https://github.com/AdguardTeam/AdguardFilters/blob/ae3a6dd44be69ab2e95dba554a2024bfe09b473e/TurkishFilter/sections/general_extensions.txt#L14